app/assets/javascripts/roster_tabs.js

```diff
@@ -10,11 +10,25 @@ function selectTab(element){
 
     studentsTab.classList.add("selected");
     unlinkedTab.classList.remove("selected");
+
+    // Unset cookie
+    document.cookie = "unlinkedSet=; expires=Thu, 18 Dec 2013 12:00:00 UTC; path=/";
   }else{
     studentsSpan.classList.add("hidden-tab");
     unlinkedSpan.classList.remove("hidden-tab");
 
     studentsTab.classList.remove("selected");
     unlinkedTab.classList.add("selected");
+
+    document.cookie = "unlinkedSet=true; path=/";
+  }
+}
+
+function setTabOnLoad(){
```

Comment:
This function runs every time we reload the partial (turbolinks, and also when a hard refresh is made on the page).